packages/gatsby-source-contentful/src/__tests__/extend-node-type.js

```diff
@@ -94,8 +95,9 @@ describe(`contentful extend node type`, () => {
     it(`generates responsive sizes data for images using all options`, async () => {
       const resp = await resolveResponsiveSizes(image, {
         maxWidth: 450,
-        maxHeight: 399,
+        maxHeight: 400,
```

Comment:
Could you change this back to 399? It was set this way to test that rounding works I believe.

Reply:
Ohh, felt so weird to me to have such an non-practical height and a resulting aspect ratio of `1.1278195488721805`

but yes, I can do that :)